torch/nn/modules/rnn.py

```diff
@@ -574,6 +574,16 @@ class LSTM(RNNBase):
         weight_hr_l[k] : the learnable projection weights of the :math:`\text{k}^{th}` layer
             of shape `(proj_size, hidden_size)`. Only present when ``proj_size > 0`` was
             specified.
+        weight_ih_l[k]_reverse: Analogous to `weight_ih_l[k]` for the backward direction.
```

Comment:
nit: `backward direction` -> `reverse direction` throughout to distinguish from backward pass terminology